### PR TITLE
Disable Labeler and all mentions of Labeling

### PR DESCRIPTION
**Pull request by Rocky43007**
For now, only. We'll re enable it once it works properly.

### CODE REVIEW

**Comment by utkubakir** [comment]
Let's disable the job as well

**Comment by Rocky43007** [comment]
> Let's disable the job as well

It should be because we just disable the `ai` feature completely?

**Comment by utkubakir** [comment]
> > Let's disable the job as well
> 
> It should be because we just disable the `ai` feature completely?

Have you tested it?

**Comment by Rocky43007** [comment]
> > > Let's disable the job as well
> > 
> > 
> > It should be because we just disable the `ai` feature completely?
> 
> Have you tested it?

Just went ahead and disabled it fully. Tested and working when adding a new folder with files that was previously unindexed.

**Comment by utkubakir** [comment]
would love to get @fogodev @HeavenVolkoff review on this

**Comment by fogodev** [comment]
For the backend I don't think that you should comment anything at all. You only need to disable the feature in the Cargo.toml files.

File: core/src/api/jobs.rs [hunk 1]
```diff
@@ -248,15 +248,17 @@ pub(crate) fn mount() -> AlphaRouter<Ctx> {
...
-						sub_path: Some(path),
-						regenerate_thumbnails: regenerate,
-						regenerate_labels: false,
-					})
-					.spawn(&node, &library)
-					.await
-					.map_err(Into::into)
+					Ok(("Currently disabled".to_string(), 0))
+
+					// Job::new(OldMediaProcessorJobInit {
```
Comment (fogodev) [inline core/src/api/jobs.rs:253]: If you comment this job, then no thumbnails will be generated lol, also no media data extraction

**Comment by brxken128** [comment]
> For the backend I don't think that you should comment anything at all. You only need to disable the feature in the Cargo.toml files. 

Yeah that seems like the easier option than commenting everything for sure.

**Review by fogodev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**